### PR TITLE
Added python code for Factorial

### LINKED ISSUE #2797: Python code for Factorial

### PR DESCRIPTION
**Pull request by Nikhita7Chavva**
Fixes #2797

#### Checklist:
- [Yes ] 4 space indentation.
- [ Yes] Coding conventions are followed.
- [ Yes] Input is taken dynamically.
- [ Yes] Sample Input / Output is added at the end of file.
- [ Yes] Logic Documentation (Comments).
- [ Yes] File names are correct.

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

#### Changes proposed in this pull request:
- I added the file called Factorial.py which is the python code for factorial

- Do not go through the file InfixtoPostfix.java as it does not take the input dyamically, I was not able to merge single file(i.e Factorial.py), both the files are getting merged by default.

#### Languages Used:
- Python
- 

#### Files Added:
- Factorial.py
- 

> We're happy to help you get this ready -- don't be afraid to ask for help.

Thanks!

### CODE REVIEW

**Comment by Nikhita7Chavva** [comment]
Ok I will do it and give u the pull requests again.
Regards,
Chavva Nikhita.

**Comment by Nikhita7Chavva** [comment]
just check it out now :)

File: Factorial/Factorial.py [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+# check if the number is negative, positive or zero
+if num < 0:
+   print("Sorry, factorial does not exist for negative numbers")
+elif num == 0:
+   print("The factorial of 0 is 1")
+else:
+   for i in range(1,num + 1):
+       factorial = factorial*i
+   print("The factorial of",num,"is",factorial)
```
Comment (MastersAbh) [inline Factorial/Factorial.py:14]: please add space after commas

File: Factorial/Factorial.py [hunk 2]
```diff
@@ -0,0 +1,21 @@
+# To take input from the user
```
Comment (MastersAbh) [inline Factorial/Factorial.py:1]: Add problem description at the top of file

File: Factorial/Factorial.py [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+elif num == 0:
+   print("The factorial of 0 is 1")
+else:
+   for i in range(1,num + 1):
+       factorial = factorial*i
+   print("The factorial of",num,"is",factorial)
+
+
+
+Output:
```
Comment (MastersAbh) [inline Factorial/Factorial.py:18]: Sample I/O should be commented out

**Comment by Nikhita7Chavva** [comment]
Ok I will do it and let you know...👍

**Comment by somya-kapoor** [comment]
Are you working on it?

**Comment by somya-kapoor** [comment]
Due to no activity for more than 7 days, I am closing this PR.

**Pull request closed without merging**